index.html

```diff
@@ -275,12 +275,12 @@
             for a subsequent requestPointerLock.
           </p>
 
-          <p>If any element (including this one) in the same document
+          <p>If any element (including this one) in the same shadow-including document
```

Comment:
Assuming the "in the same shadow-including document" [PR#273](https://github.com/whatwg/dom/issues/273) will be accepted, can I leave this part as is?